_data/other.yml

```diff
@@ -252,6 +252,13 @@ websites:
       twitter: indeed
       facebook: Indeed
 
+    - name: Indiegogo
+      url: https://www.indiegogo.com/
+      twitter: Indiegogo
```

Comment:
Doesn't look like this account is monitored. What do you think about using @IndiegogoUK instead?